src/test/java/dog/pawbook/logic/commands/DeleteProgramCommandTest.java

```diff
@@ -0,0 +1,4 @@
+package dog.pawbook.logic.commands;
+
+public class DeleteProgramCommandTest {
+}
```

Comment:
leave out the file if not written so others can easily tell if the test has been written